### ISSUE TITLE
source-maps at the browser

### ISSUE THREAD

**Issue by cescoferraro** [visionmedia/debug#385]
Is this the default behaviour? No way I could get the correct/(the one I want) line number?

```
debug.enable("cesco");
debug("cesco")("HELLO");
console.log("hello");
// outputs
cesco HELLO +0ms  browser.js:123
hello   index.tsx:14
```

**Comment by thebigredgeek** [comment]
We aren't currently transpiling, so not sure.  It sounds like your own source maps aren't working after minifying debug along with your vendor libs?

**Comment by thebigredgeek** [comment]
@cescoferraro could you further elaborate on your environment/setup?  If not I am going to close this for now :(

**Comment by cescoferraro** [comment]
@thebigredgeek I will be working on a different project this week. But it uses react & webpack2 2.2.0-rc.1, so there might me something going on there or with my configuration. 

Anyway. my source maps as working because it to takes me to the right .tsx file where I called console.log. The same thing does not happen with debug, which shows https://github.com/visionmedia/debug/blob/master/src/browser.js#L123 

It this supposed to happen? We are migrating from console.log to debug, but there is no use for us if we cant figure out we the caller at. Maybe its something wrong with this IE workaround. I used to work on a angular2 project, as as far as I can remeber, this used to work out of the box.

>   // this hackery is required for IE8/9, where
>   // the `console.log` function doesn't have 'apply'

I will try a couple of other things when I get the chance to work there again. Worst case scenario , it gonna be on manday. Thanks for your time!
